src/pocketmine/entity/Living.php

```diff
@@ -232,6 +232,14 @@ public function getEffect(int $effectId){
 	public function hasEffect(int $effectId) : bool{
 		return isset($this->effects[$effectId]);
 	}
+	
+	/**
+	 * Returns whether the any effect is active on the mob.
```

Comment:
Grammar?

Reply:
@Muqsit I'd appreciate it if you fixed my mistake. I don't know english well.

Reply:
Returns whether the mob has any active effects.